bin/yb-ctl

```diff
@@ -403,6 +421,9 @@ class ClusterOptions:
 
         self.installation_dir = None
         yb_src_root_candidate = os.path.dirname(os.path.dirname(os.path.dirname(self.script_dir)))
+        # For internal development, assume $PARENT_DIR/yugabyte-installation/bin/yb-ctl will have
```

Comment:
internal development -> YugaByte DB development (this workflow will also apply to all contributors to YugaByte DB).